.travis.yml

```diff
@@ -14,9 +14,12 @@ notifications:
 
 install:
   - go get -u github.com/Masterminds/glide
+  - rm -rf vendor/
```

Comment:
It's a test on `.travis` to be sure we didn't sneak any dev changes to vendor dir.
Take a look down below:
```
- git diff --exit-code
```
https://github.com/percona/pmm-client/pull/27/files/149e76875da5d49119b3f675f45f548e632d2b89#diff-354f30a63fb0907d4ad57269548329e3R21

We can add later better comments.